docs/source/pipelines.rst

```diff
@@ -41,7 +41,8 @@ as follows:
 
       pip install mlflow[pipelines]
 
-  Note that MLflow Pipelines requires Make, which may not be preinstalled on some Windows systems.
+  Note that MLflow Pipelines requires [Make](https://www.gnu.org/software/make/),
+  which may not be preinstalled on some Windows systems.
   Windows users must install Make before using MLflow Pipelines. For more information about
   installing Make on Windows, see https://gnuwin32.sourceforge.net/install.html.
```

Comment:
While we're here, can we add an extra newline between the next sentence (line 47) and the databricks section (line 49)? Didn't realize, but the text is mashed together a bit:

![Uploading Screen Shot 2022-10-20 at 5.49.29 PM.png…]()

Reply:
Got it. Can you try https://stackoverflow.com/a/46528727/11952869?

Reply:
Adding a `|` in a new line doesn't work: 
`Bullet list ends without a blank line; unexpected unindent.`
Or simply adding a real `|` to the end of the first bullet.

Reply:
Dumb question but a newline is two spaces in markdown, is it the same here?

Reply:
Tried. No effect.

Reply:
Can you look into this further and resolve before merging? Perhaps the solutions in https://stackoverflow.com/questions/54001054/force-a-new-line-in-sphinx-text will work.